aries-site/src/examples/components/controls/ButtonExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Box, Button } from 'grommet';
+import { Next } from 'grommet-icons';
+
+export const ButtonExample = () => {
+  return (
+    <Box direction="column" align="start" gap="large">
+      <Button
+        label="Check it out!"
+        icon={<Next />}
+        reverse
+        primary
```

Comment:
I think we said we should have one or two instances of `primary` Button, I think the ButtonExample may be used more than a couple of times.